static/js/containers/CourseSearchPage.js

```diff
@@ -210,73 +211,76 @@ export class CourseSearchPage extends React.Component<Props, State> {
+                    <Checkbox
+                      name="topics"
+                      value={topic}
+                      checked={R.contains(topic, topics || [])}
+                      onChange={this.onUpdateFacets}
+                    >
+                      {topic}
+                    </Checkbox>
+                  </div>
+                ))}
+              </div>
 
-            <div className="course-facets">
-              <div className="facet-title divider">Availability</div>
-              {COURSE_AVAILABILITIES.map((availability, i) => (
-                <div key={i}>
-                  <Checkbox
-                    name="availabilities"
-                    value={availability}
-                    checked={R.contains(availability, availabilities || [])}
-                    onChange={this.onUpdateFacets}
-                  >
-                    {_.capitalize(availability)}
-                  </Checkbox>
-                </div>
-              ))}
-            </div>
-            <div className="course-facets">
-              <div className="facet-title divider">Platforms</div>
-              {_.sortBy(facetChoices.platforms).map((platform, i) => (
-                <div key={i}>
-                  <Checkbox
-                    name="platforms"
-                    value={platform}
-                    checked={R.contains(platform, platforms || [])}
-                    onChange={this.onUpdateFacets}
-                  >
-                    {_.upperCase(platform)}
-                  </Checkbox>
-                </div>
-              ))}
-            </div>
-          </Card>
-        </Cell>
-        <Cell width={8}>{error ? null : this.renderResults()}</Cell>
-      </Grid>
+              <div className="course-facets">
+                <div className="facet-title divider">Availability</div>
+                {COURSE_AVAILABILITIES.map((availability, i) => (
+                  <div key={i}>
+                    <Checkbox
+                      name="availabilities"
+                      value={availability}
+                      checked={R.contains(availability, availabilities || [])}
+                      onChange={this.onUpdateFacets}
+                    >
+                      {_.capitalize(availability)}
+                    </Checkbox>
+                  </div>
+                ))}
+              </div>
+              <div className="course-facets">
+                <div className="facet-title divider">Platforms</div>
+                {_.sortBy(facetChoices.platforms).map((platform, i) => (
+                  <div key={i}>
+                    <Checkbox
+                      name="platforms"
+                      value={platform}
+                      checked={R.contains(platform, platforms || [])}
+                      onChange={this.onUpdateFacets}
+                    >
+                      {_.upperCase(platform)}
+                    </Checkbox>
+                  </div>
+                ))}
```

Comment:
this `Checkbox` creating map seems similar to the one above (on line 234), do you think they could be factored out into a shared method?

Reply:
✔️